pkg/icingadb/history/sync.go

```diff
@@ -2,12 +2,15 @@ package history
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
Let me guess: it makes no sense to review the diff as such, but just the new file?

Reply:
Yes.